vng_api_common/middleware.py

```diff
@@ -234,31 +234,45 @@ def client_id(self) -> Union[str, None]:
 
         return client_id
 
-    def has_auth(self, scopes: List[str], zaaktype: Union[str, None],
-                 vertrouwelijkheidaanduiding: Union[str, None]) -> bool:
+    def filter_vertrouwelijkheidaanduiding(self, base: QuerySet, value) -> QuerySet:
+        if value is None:
+            return base
 
+        order_provided = VertrouwelijkheidsAanduiding.get_choice(value).order
+        order_case = VertrouwelijkheidsAanduiding.get_order_expression("max_vertrouwelijkheidaanduiding")
+
+        return base.annotate(max_vertr=order_case).filter(max_vertr__gte=order_provided)
```

Comment:
took me a while to wrap my head around it that it's `__gte` instead of `__lte`, can you add a comment?

we're only considering authorizations here that have a `max_vertrouwelijkheidaanduiding` bigger or equal than what we're checking for the object.

In the situation where you'd be filtering data to limit collections, it's the other way around. It's correct, but it looked weird, so a clarification is good!

Reply:
A comment is added to the code